### PR TITLE
ci: add labeler config

### PR DESCRIPTION
**Pull request by Baroshem**
This PR adds labeler workflow for VSF 2 (Next) to automatically assing labels based on modified files.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Add 'VSF1' label to any change to packages files
+VSF1:
+- true
+
+# Add 'core' label to any change to packages/core files within the source dir EXCEPT for the docs sub-folder
```
Comment (Fifciu) [inline .github/labeler.yml:5]: `packages/core` -> `core`

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+core:
+- core/**/*
+
+# Add 'commercetools' label to any change to packages/commercetools files
+commercetools:
```
Comment (Fifciu) [inline .github/labeler.yml:10]: It does not exist in vsf1

**Review by Fifciu — CHANGES_REQUESTED** [review]
Awesome one. Just one thing to update. It would be awesome if you could also prepare similar PR for hotfix 1.12.3 and develop

File: .github/labeler.yml (hunk 1 above)
Comment (Baroshem) [reply .github/labeler.yml]: You are right. Changed it. Thanks!

File: .github/labeler.yml (hunk 2 above)
Comment (Baroshem) [reply .github/labeler.yml]: Removed

**Comment by bloodf** [comment]
This is closed due to inactivity.

If you want your PR to be re-opened, please update your files, check for any conflicts.

If your PR is for Vue Storefront 1 target the new repository at https://github.com/vuestorefront/vue-storefront-1

**Pull request closed without merging**